app/models/image.rb

```diff
@@ -68,11 +70,19 @@ def imageable_class
 
     def validate_image_dimensions
```

Comment:
Let's validate the dimensions in the model that we use to image, no in the image model.